src/vs/code/electron-main/window.ts

```diff
@@ -334,6 +335,22 @@ export class VSCodeWindow {
 		return this._readyState;
 	}
 
+	private registerNavigationListenerOn(command: string, back: string, forward: string) {
+		this._win.on(command, (e, cmd) => {
+			if (this.readyState !== ReadyState.READY) {
+				return; // window must be ready
+			}
+
+			if (cmd === back) {
+				this.send('vscode:runAction', 'workbench.action.navigateBack');
+			} else if (cmd === forward) {
+				this.send('vscode:runAction', 'workbench.action.navigateForward');
+			} else {
+				console.warn('[electron command]: did not handle: ', command);
```

Comment:
@bpasero We can't predict all the available commands for every event related to navigation that exists. You would have to maintain a list somewhere to keep them up to date. 
E.g for `swipe` it's `left` and `right` but on `app-command` it's:  `browser-backward` and `browser-forward`